### PR TITLE
Few issues found when testing

### PR DESCRIPTION
**Pull request by devashishdxt**
1. `register_delegate_keys` usage string is not in sync with actual usage of cli command.
2. Fee with incorrect denomination is passed in client `cmd_cosmos_to_eth`.

### CODE REVIEW

File: module/go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/althea-net/peggy/module
 go 1.15
 
 require (
-	github.com/cosmos/cosmos-sdk v0.40.0-rc7
+	github.com/cosmos/cosmos-sdk v0.40.0
```
Comment (jkilpatr) [inline module/go.mod:6]: this is good

File: orchestrator/client/src/main.rs [hunk 2]
```diff
@@ -103,10 +103,14 @@ async fn main() {
             amount,
             denom: peggy_denom.clone(),
         };
+        let bridge_fee = Coin {
```
Comment (jkilpatr) [inline orchestrator/client/src/main.rs:106]: Good catch here, could you remove the user fee argument from the cosmos-to-eth string above?

**Review by jkilpatr — CHANGES_REQUESTED** [review]
This looks great! thanks for taking the time to test and open a pull request.

If you could make a tiny change to your second commit I can land this right away.

Please let me know if you don't have time and I'll be happy to do it myself and then merge this pr

File: orchestrator/client/src/main.rs (hunk 2 above)
Comment (devashishdxt) [reply orchestrator/client/src/main.rs:106]: You mean eth-to-cosmos? Code uses fee argument for cosmos-to-eth.

Comment (jkilpatr) [reply orchestrator/client/src/main.rs:106]: yeha I see in the batch request. I wonder if it's worth keeping put the pull request at least makes sense. I'm going to merge these. That's reasonable to clean up later. Thanks for your patience and contribution!

**Review by jkilpatr — APPROVED** [review]

**Pull request merged**